pkg/api/meta/meta.go

```diff
@@ -104,13 +104,13 @@ func TypeAccessor(obj interface{}) (TypeInterface, error) {
 // or manipulate resource version on objects derived from core API
 // metadata concepts.
 func NewAccessor() MetadataAccessor {
-	return resourceAccessor{}
+	return ResourceAccessor{}
```

Comment:
Because package restmapper(pkg/api/rest/restmapper/restmapper_test.go) will use it. :)